backend/controllers/users.js

```diff
@@ -4,7 +4,6 @@ const jwt = require("jsonwebtoken");
 const crypto = require("crypto");
 const bcrypt = require("bcrypt");
 const { validationResult } = require("express-validator");
-require("dotenv").config();
```

Comment:
I removed this since we only need the dotenv require once inside the backend application (inside `index.js`)